skyfield/nutationlib.py

```diff
@@ -29,6 +29,32 @@ def compute_nutation(t):
                    cpsi * cobm * sobt - sobm * cobt,
                    cpsi * sobm * sobt + cobm * cobt)))
 
+def compute_ECLIP(t):
+    """Generate the ecliptic rotations for Time `t`.
+
+    If the Julian date is scalar, a simple ``(3, 3)`` matrix is
+    returned; if the date is an array of length ``n``, then an array of
+    matrices is returned with dimensions ``(3, 3, n)``.
+
+    """
+    oblm, oblt, eqeq, psi, eps = earth_tilt(t)
```

Comment:
Oh, wow! I hadn't been thinking about ecliptic calculations when I adapted `earth_tilt()` to Skyfield and so hadn't realized that it provided the exact angle needed for the ecliptic transform.

How did you choose between `oblm` and `oblt`?

Reply:
I chose the one that gives the same result as novas.